### PR TITLE
add requested metadata fields

### PR DESCRIPTION
**Pull request by andre-urbani**
### What

Add the additional requested metadata fields as specified in Trello ticket
Trello - https://trello.com/c/V5Nffufg/5900-update-xls-meta-data-tab-to-include-additional-meta-data-fields

### How to review

Confirm changes make sense and match the changes requested in Trello ticket

### Who can review

Anyone

### CODE REVIEW

File: handler/metadata.go [hunk 1]
```diff
@@ -141,6 +144,36 @@ func (h *XlsxCreate) AddMetaDataToExcelStructure(ctx context.Context, excelInMem
 
 	processMetaElement("Version", strconv.Itoa(meta.Version.Version), true)
 	processMetaElement("Dataset version", meta.DatasetLinks.LatestVersion.URL, true)
+	processMetaElement("Statistical Disclosure Control Statement", sdcStatement, true)
+	processMetaElement("Area Type", areaTypeStatic, true)
+
+	for _, dimensions := range meta.Version.Dimensions {
+		if dimensions.IsAreaType == &trueVal {
```
Comment (j-s-rawat) [inline handler/metadata.go:151]: Won't the correct test will be to compare the value of  IsAreaType 
i.e. *dimensions.IsAreaType ==  trueVal

File: handler/metadata.go [hunk 2]
```diff
@@ -141,6 +144,36 @@ func (h *XlsxCreate) AddMetaDataToExcelStructure(ctx context.Context, excelInMem
...
+			processMetaElement("Area Type Name", dimensions.Label, true)
+			processMetaElement("Area Type Description", dimensions.Description, true)
+		}
+
+		if dimensions.IsAreaType == &falseVal {
```
Comment (j-s-rawat) [inline handler/metadata.go:156]: Similar to above

File: handler/metadata.go [hunk 3]
```diff
@@ -141,6 +144,36 @@ func (h *XlsxCreate) AddMetaDataToExcelStructure(ctx context.Context, excelInMem
...
+
+	processMetaElement("Coverage", coverageStatic, true)
+
+	if meta.DatasetDetails.VersionsList.Items != nil {
+		formatToParse := "2006-01-02T15:04:05.000Z"
+		rowNumber++
+		processMetaElement("Version History", "", false)
+		for _, versions := range meta.DatasetDetails.VersionsList.Items {
+			processMetaElement("Version Number", strconv.Itoa(versions.Version), true)
+			date, _ := time.Parse(formatToParse, versions.ReleaseDate)
```
Comment (j-s-rawat) [inline handler/metadata.go:170]: better not to ignore error and add a condition as in case of error as default date will be returned in case of error

File: handler/metadata.go [hunk 4]
```diff
@@ -140,7 +149,73 @@ func (h *XlsxCreate) AddMetaDataToExcelStructure(ctx context.Context, excelInMem
...
+
+			if *versions.Alerts != nil {
+				for _, alerts := range *versions.Alerts {
+					processMetaElement("Reason for New Version", alerts.Description, true)
+				}
+			}
+		}
+	}
+
+	if meta != nil && meta.DatasetDetails.RelatedContent != nil {
```
Comment (j-s-rawat) [inline handler/metadata.go:208]: we might not need to test meta != nil as it has already been used by the time code execution reaches here

**Review by thomasridd — APPROVED** [review]
